pkg/asset/manifests/template.go

```diff
@@ -17,6 +17,20 @@ type AzureCredsSecretData struct {
 	Base64encodeRegion         string
 }
 
+// GCPCredsSecretData holds encoded credentials and is used to generate cloud-creds secret
+type GCPCredsSecretData struct {
```

Comment:
the GCP creds are the service account json file as it.. not individual fields from the serviceaccount json.